### ISSUE TITLE
Sign in With Discord

### ISSUE THREAD

**Issue by IntelliJinceTech** [ImanKahlila/ProjectHunt#56]
It remains to be seen how the authorization changes will go due to the widespread username changes that are coming to Discord. Note - it is coming in waves so the changes that we make for this issue might possibly need to be backward compatible. 

More discussion is needed.

**Comment by ImanKahlila** [comment]
@andrewquach-dev @ThomasCarney315 What are you guys' thoughts on this?

**Comment by andrewquach-dev** [comment]
I feel like this is a low-priority task because we already have email login and the website works fine without it. But I know that 100devs students mostly use Discord so it would be nice to have. Also, it'd be a great learning experience reading Discord's API documentation and implementing it to the project.

**Comment by ThomasCarney315** [comment]
Agreed. Definitely, something that would be worth researching in the future. It may require an update to  the passport dependency or more.
